src/scipp/spatial/__init__.py

```diff
@@ -2,4 +2,8 @@
 # Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
 # @author Simon Heybrock
 # flake8: noqa
-from .transform import from_rotvec, as_rotvec
+from . import transform
```

Comment:
Can you elaborate why we import `transform`, but then functions from `transform`?